src/Site.js

```diff
@@ -104,6 +113,24 @@ const SITE_NAV_DEFAULT = '<navigation>\n'
   + '* [Home :glyphicon-home:]({{baseUrl}}/index.html)\n'
   + '</navigation>\n';
 
+const TOP_NAV_DEFAULT = '<navbar placement="top" type="inverse">\n'
+  + '  <a slot="brand" href="{{baseUrl}}/index.html" title="Home" class="navbar-brand">'
+  + '<i class="far fa-file-image"></i></a>\n'
+  + '  <li><a href="{{baseUrl}}/index.html" class="nav-link">HOME</a></li>\n'
+  + '  <li><a href="{{baseUrl}}/about.html" class="nav-link">ABOUT</a></li>\n'
+  + '  <li>\n'
+  + '    <a href="https://github.com/MarkBind/markbind" target="_blank" class="nav-link">\n'
```

Comment:
The github link seem unrelated. Is there a way to infer the wiki's code repository?

Reply:
> The github link seem unrelated. Is there a way to infer the wiki's code repository?

I'm not sure how to go about implementing this actually, should I let the user specify the Github URL as a parameter for the `--convert` command?

Reply:
Usually the wiki has a `.wiki` attached to the end. I'm thinking we could get the git remote and parse out the actual repository's url. This may require an external library though.

> should I let the user specify the Github URL as a parameter for the --convert command

I think its a bit weird to open an option just for that. Also what happens if they do not specify it? Alternatively i think we could write a comment and have the link point to MarkBind or we could just remove the link.

@Chng-Zhi-Xuan any thoughts?

Reply:
If there's no easy way of deducing the actual repository's URL, it is OK to not include this.